compactor/src/compact.rs

```diff
@@ -273,6 +287,18 @@ impl Compactor {
             "Compact partition duration",
         );
 
+        let compaction_query_highest_throughput_duration: Metric<DurationHistogram> = registry
+            .register_metric(
+                "compaction_query_highest_throughput_duration",
+                "Duration of a catalog query that gets the most recent highest ingested throughput partitions",
+            );
+
+        let compaction_query_most_l0_duration: Metric<DurationHistogram> = registry
+            .register_metric(
+                "compaction_query_most_l0__duration",
```

Comment:
It looks like there's an extra underscore in this metric name? 

```suggestion
                "compaction_query_most_l0_duration",
```